.whitelist.gpg

```diff
@@ -56,4 +56,56 @@ xRYB7YaXneBhWc7F6cC0mz4mwrcVH952VR9qAKVXK5QR16EZaDumxjB2uH619w68
 oekxZ1I7S/t/zqeQ04vucdhRVYyzyfusjtmwSOLkZDdLfeFxqkB7vnns+yGqkLfq
 WkotIE5Gq09SL6AXJAVvq9osefk4kDYc5zXmFuA1CHpQJMBziWfOC1c7RndBzw==
 =KI3J
------END PGP PUBLIC KEY BLOCK-----
\ No newline at end of file
+-----END PGP PUBLIC KEY BLOCK-----
```

Comment:
I will remove my gpg key before the final review and merge, but it is necessary for testing.

Reply:
I don't have a problem with your key being on the whitelist, myself.